src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,257 @@
+#Contract Entities
+####AuditableEntity
+#####What is it?
+`AuditableEntity` is the abstract base type for all OpenHDS entities. The class provides fields useful for ‘auditing’ or keeping track of meta information pertaining to the concrete type like when it was created and last modified.

```

Comment:
I think UuidIdentifiable is the base for all OpenHDS entities.  There are important things like ProjectCodes and Roles which are not AuditableEntities.